src/radical/pilot/agent/launch_method/jsrun.py

```diff
@@ -160,12 +161,16 @@ def get_launch_cmds(self, task, exec_path):
                 self._create_resource_set_file(slots, uid, sbox)
 
         else:
+            # for OpenMP threads a corresponding parameter should be provided
+            # in task description - `td.threading_type = rp.OpenMP`,
+            # and RP will set `export OMP_NUM_THREADS=<cores_per_rank>`
+            cores_per_rs = math.ceil(
+                td['cores_per_rank'] / self._rm_info['threads_per_core'])
             # -b: bind to RS
```

Comment:
Can we explain what is `RS: Resource Sets` in here? So if someone reads the code can understand the acronym.